### ISSUE TITLE
Feature request: Allow rotating the axes for scroll wheel emulation

### ISSUE THREAD

**Issue by NiklausHofer** [riverwm/river#958]
I was pleasantly surprised to find that river has support for scroll wheel emulation through the `riverctl input <input> scroll-method button` and `riverctl input <input> scroll-button <button>` commands.

The only thing I'm missing is the possibility to rotate the axes. I use that on my Logitech track ball to make scrolling up/down less agitating.

On X11 I use the below command. Then, instead of moving the ball back and forth, I can move it sideways instead to scroll up/down.

```bash
awful.util.spawn_with_shell("xinput --set-prop 'Logitech USB Receiver Mouse' 'Evdev Wheel Emulation Axes' 5 4 0 0")
```

Would be great to see something like that in river, too!

**Comment by Leon-Plickat** [comment]
That sounds like something that should be implemented in libinput.

**Comment by ifreund** [comment]
True, it wouldn't feel out of place as a libinput configuration option. Opening a feature request on the libinput bug tracker certainly couldn't hurt: https://gitlab.freedesktop.org/libinput/libinput/-/issues

On the other hand, there is nothing stopping us from implementing this horizontal/vertical flip in river assuming I understand the feature correctly.

**Comment by novakne** [comment]
How about this in libiput? 

https://wayland.freedesktop.org/libinput/doc/latest/configuration.html#rotation
https://wayland.freedesktop.org/libinput/doc/1.24.0/api/group__config.html#ga136b2f7753235c5d2a9fb2762a040eef

**Comment by ifreund** [comment]
> How about this in libinput?
> 
> https://wayland.freedesktop.org/libinput/doc/latest/configuration.html#rotation https://wayland.freedesktop.org/libinput/doc/1.24.0/api/group__config.html#ga136b2f7753235c5d2a9fb2762a040eef

I haven't tested, but it looks like that applies to more than just scroll axes?

**Comment by novakne** [comment]
Ah right, that's probably change everything

**Comment by NiklausHofer** [comment]
> True, it wouldn't feel out of place as a libinput configuration option. Opening a feature request on the libinput bug tracker certainly couldn't hurt: https://gitlab.freedesktop.org/libinput/libinput/-/issues

Opened a bug on libinput: https://gitlab.freedesktop.org/libinput/libinput/-/issues/965
